extensions/php/package.nls.json

```diff
@@ -1,6 +1,7 @@
 ﻿{
-	"configuration.validate.enable": "Whether php validation is enabled or not.",
-	"configuration.validate.executablePath": "Points to the php executable.",
+	"configuration.builtInCompletions.enable": "Enable/disable built-in PHP completions.",
+	"configuration.validate.enable": "Enable/disable built-in PHP validation.",
+	"configuration.validate.executablePath": "Points to the PHP executable.",
```

Comment:
Do we have a strategy for aliasing settings names? e.g. php.validate.enable is somewhat ambiguous, but it's also already in use.